CMakeLists.txt

```diff
@@ -938,6 +939,8 @@ elseif(${PLAYER_TARGET_PLATFORM} MATCHES "^(3ds|switch|psvita)$")
 		ctr_create_3dsx(easyrpg-player
 			SMDH easyrpg-player.smdh)
 	elseif(NINTENDO_SWITCH)
+		target_include_directories(${PROJECT_NAME} PUBLIC
```

Comment:
The toolchain only sets ``-L`` but not ``-I``.
The proper way would be to request all the libraries through pkg-config (as we can be certain pkg-config is available due to devkitpro). That was the first thing I tried but CMake dislikes the pc files because some of them contain ``prefix=$(devkitpro)...`` instead of ``prefix=/opt/devkitpro``.

So would require asking DKP maintainers to patch all the pc files :/

Reply:
Actually there is a CMake config file for use with ``find_package(OpenGL CONFIG REQUIRED`` but the config file is not found because it is not in one of the search directories used by CMake.

I'm not sure if this is a CMake bug or not.

I asked for clarification here: https://gitlab.kitware.com/cmake/cmake/-/issues/23192

Reply:
The ``find_package`` problem was resolved in the recent dkp cmake update. Solving the last blocker.

Reply:
Interestingly enough it works locally, but not on my ports jenkins agent.